tripy/examples/diffusion/compare_images.py

```diff
@@ -0,0 +1,142 @@
+#
+# SPDX-FileCopyrightText: Copyright (c) 2025-2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import os
+import numpy as np
+from PIL import Image
+from skimage.metrics import structural_similarity
+import glob
+
+
+def load_reference_image(image_path, verbose=False):
+    """Load reference image from file path."""
+    if not os.path.exists(image_path):
+        raise FileNotFoundError(f"Reference image not found: {image_path}")
+
+    if verbose:
+        print(f"[I] Loading reference image from {image_path}")
```

Comment:
Doesn't have to be done in this PR, but I don't think these prints are so verbose that they need to be hidden behind a flag. IMO we could probably just drop the verbose option and make all of these unconditional.